libs/template/templates/default-python/template/{{.project_name}}/src/dlt_pipeline.ipynb.tmpl

```diff
@@ -35,7 +35,6 @@
     "# Import DLT and src/{{.project_name}}\n",
     "import dlt\n",
     "import sys\n",
-    "sys.path.append(spark.conf.get(\"bundle.sourcePath\", \".\"))\n",
```

Comment:
We need to verify the PYTHONPATH fix for DLT has been rolled out everywhere before making this change.

Reply:
cc @justintang-db

Reply:
LGTM. DLT side change is in dlt release 37 and has been rolled out to all stages yesterday.